src/modV.js

```diff
@@ -1,53 +1,59 @@
 const EventEmitter2 = require('eventemitter2').EventEmitter2;
 const shaderInit = require('./shader-env');
 const threeInit = require('./three-env');
+const Layer = require('./Layer');
 const MM = require('./media-manager');
 require('./fragments/array-contains');
 require('script-loader!../libraries/beatdetektor.js');
 
+
 /**
  * modV
  * @extends {EventEmitter2}
  */
-class modV extends EventEmitter2 {
+class ModV extends EventEmitter2 {
```

Comment:
But it's just a class name, and in `app.js` you can see line: `const modV = require('./modV');`.
What's wrong?